src/systems.rs

```diff
@@ -159,7 +170,7 @@ pub fn movement(
         let mut jump = if controller.jump_timer > 0.0 && !grounded {
             if !input.jumping {
                 controller.jump_timer = 0.0;
-                velocity.linvel.project_onto(settings.up_vector) * -settings.jump_stop_force
+                velocity.linvel.project_onto(settings.up_vector) * -settings.jump_stop_force * dt
```

Comment:
This is a one-frame force to immediately cancel jump velocity, it's not supposed to be multiplied by dt
```suggestion
                velocity.linvel.project_onto(settings.up_vector) * -settings.jump_stop_force
```